applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast.h

```diff
@@ -38,7 +38,7 @@
 					   BT_AUDIO_CONTEXT_TYPE_MEDIA)
 
 #else
-#error Unsupported LC3 codec preset for broadcast
+/* #error Unsupported LC3 codec preset for broadcast */
```

Comment:
I see the point, since broadcast.h will be included but there's no configuration for broadcasting which trigger the compile error. Should we just remove this?

Reply:
Yes, let's have a look at this in the broader scope once all four PRs have been merged to the "daughtership".